Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -72,76 +72,76 @@
                                   CommandParameter="{Binding SelectedItem, Mode=OneWay}"
                                   Visibility="{Binding VB6Visibility}">
                     <MenuItem.Icon>
-                        <Image Source="{StaticResource AddTestModuleImage}" />
+                        <Image Height="16" Source="{StaticResource AddTestModuleImage}" />
```

Comment:
does this work for hidpi monitors correctly?

Reply:
I don't know. I don't have that style of monitor I can test with.

I made an assumption that since another icon had its size limited, https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml#L171, adding it to the others would be fine.